study-builder/studies/rarex/snippets/dob-child-required.conf

```diff
@@ -1,5 +1,6 @@
 {
```

Comment:
This one is used in both parental-consent and parental-assent:

![Screen Shot 2021-05-11 at 11 15 12 AM](https://user-images.githubusercontent.com/5713833/117847512-66fe4600-b250-11eb-881c-33a1c9aaca90.png)